public/pages/createDetector/containers/CreateDetector.tsx

```diff
@@ -118,6 +115,8 @@ export function CreateDetector(props: CreateADProps) {
   };
 
   const handleSubmit = async (values: ADFormikValues, formikBag: any) => {
+    console.log('hello');
+    console.log(values);
```

Comment:
nit: can remove those lines for debug use

Reply:
removed.  Thanks for the catch.